### ISSUE TITLE
Font Taxonomy

### ISSUE THREAD

**Issue by xfq** [w3cping/font-fp-protection-browser-hinting#1]
The classification in https://github.com/w3cping/font-fp-protection-browser-hinting#terms may cause ambiguity because it is different from the taxonomy in CSS here: https://drafts.csswg.org/css-fonts-4/#font-taxonomy

Here's a correspondence table:

| PING           | CSSWG                |
|----------------|----------------------|
| OS Fonts       | Preinstalled Fonts   |
| System Fonts   | Installed Fonts      |
| Web Fonts      | Web Fonts            |
| Hinted Fonts   | N/A                  |
| Selected Fonts | N/A                  |
| Page Fonts     | N/A                  |
| N/A            | User-Installed Fonts |
| N/A            | System Font          |

**Comment by astearns** [comment]
I think new categories are fine, but matching the names/indicating the correspondences would be good. Also, it would be better to change 'hinted' to something else, since hinting is already used in the font world for something entirely different. Perhaps `UA-selected` or `required language support`?

**Comment by svgeesus** [comment]
I agree that matching with the terms [already defined in Fonts 4](https://drafts.csswg.org/css-fonts-4/#font-taxonomy) would be good. For new categories, I would be happy adding them to Fonts 4 as well.
